samples_project/Assets/StarterAssets/Editor/StarterAssetsDeployMenu.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Linq;
+using StarterAssetsPackageChecker;
```

Comment:
This line was causing an error in my script when I loaded it. 

This starterasset is quite heavy and it is bringing a lot into our repo and adding a lot of dependencies